ansible/inventory/hosts.yml

```
@@ -0,0 +1,74 @@
+---
+all:
+  vars:
+    # Project configuration
+    project_name: education-website
+    project_user: "{{ ansible_user }}"
+    project_path: "/home/{{ project_user }}/{{ project_name }}"
+    python_version: "3.11"
+    app_port: 8000
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)